test-d/foundry/hooks.test-d.ts

```diff
@@ -0,0 +1,7 @@
+import '../../../../index';
+
+Hooks.on('applyActiveEffect', (actor: Actor, change: ActiveEffect.Change) => console.log('foo'));
```

Comment:
Can we get rid of the lint errors somehow? maybe by not adding explicit type annotations to the parameters and calling `expectType` on the parameters in the arrow functions?